content/sips/sip-420.md

```diff
@@ -0,0 +1,105 @@
+
+This SIP proposes a mechanism for SNX stakers to provide staked SNX to protocol-owned staking pool, where the debt will be owned by the protocol and used to generate yield via Ethena. This effectively reduces the complexity of staking for individual stakers while making the overall system more capital efficient.
+
+## Abstract
+
+<!--A short (~200 word) description of the proposed change, the abstract should clearly describe the proposed change. This is what *will* be done if the SIP is implemented, not *why* it should be done or *how* it will be done.-->
+
+This SIP proposes the creation of a protocol-owned debt pool, effectively enabling “delegated staking.” SNX holders can contribute their existing debt positions to this pool, which can support a lower issuance ratio (e.g., 200%) because there is no concern about bad debtors within the pool. Over twelve months, debt positions (but not the collateral) will be transferred to this new staking pool. The protocol will use its newly minted sUSD to mint sUSDe (via the Ethena protocol), providing additional yield. Meanwhile, a higher c-ratio (1000%) will be enforced for solo stakers, incentivizing them to join the delegated system. The collateral from the treasury staking position will be redistributed to all participating stakers over a multi-year period, and an sUSD/USDe liquidity pool will be incentivized for the liquidity of the stablecoin. Finally, 250k SNX from the treasury will be allocated as a bonus for the core contributors who implement this proposal at the council's discretion.
+
+## Motivation
+
+<!--This is the problem statement. This is the *why* of the SIP. It should clearly explain *why* the current state of the protocol is inadequate.-->
+
+No inflation and mediocre fees have created minimal incentive for SNX staking, even with the lower risk in the current system. Staking rates are at multi-year lows, and a large minority of SNX is unstaked and sitting idle on exchanges. This proposal removes the need for active individual staking and creates a capital-efficient structure that significantly enhances incentives. It also helps rectify past debt inflation issues by ensuring only willing participants remain in the protocol’s debt pool.
+
+## Specification
+
+<!--There are five sections in the Specification:
+1. Overview
+2. Rationale
+3. Technical Specification
+4. Test Cases
+5. Configurable Values (Via SCCP)
+-->
+
+### Overview
+
+<!--This is a high level overview of how the SIP will solve the problem.-->
+
+1. **Create a delegated staking contract**  
+   - Stakers deposit their existing debt positions into the new pool.
+2. **Raise the current c-ratio to 1000%**  
+   - Discourage solo staking and encourage delegation to the pool.
+3. **Set the issuance ratio for the delegated pool to 200%**  
```

Comment:
in previous proposals I was under the understanding that stakers would still be able to mint up to th ecurrent c-ratio (500%) of debt as effectively a "loan" against the spartan council. This also allows for a very natural handling of existing outstanding debt that current stakers face. Are we looking into phasing this out?

Reply:
That was one of the earlier options, but IMO it is too confusing and means that you still have the risk of bad debt in the system. The solution now is to allow for solo staking to happen in parallel for some period of time.